src/phases/attempt-capture-phase.ts

```diff
@@ -81,7 +81,7 @@ export class AttemptCapturePhase extends PokemonPhase {
     this.pokeball.setOrigin(0.5, 0.625);
     field.add(this.pokeball);
 
-    globalScene.audioManager.playSound("se/pb_throw", isCritical ? { rate: 0.2 } : undefined); // Crit catch throws are higher pitched
+    globalScene.audioManager.playSound(isCritical ? "se/crit_throw" : "se/pb_throw");
```

Comment:
NIT: not sure if that makes it better or worse. I'll let you and allt he other reviewers judge
```suggestion
    globalScene.audioManager.playSound(`se/${isCritical ? "crit" : "pb"}_throw`);
```

Reply:
My personal opinion is that there's no need to inline it.
```ts
if (isCritical) {
  globalScene.audioManager.playSound("se/pb_throw");
} else {
  globalScene.audioManager.playSound("se/crit_throw");
}
```

Reply:
3 Times the charm:
```suggestion
    let throwSound: string;
    
    if (isCritical) {
      throwSound = "se/crit_throw";
    } else {
      throwSound = "se/pb_throw";
    }
   
    globalScene.audioManager.playSound(throwSound);
```